### ISSUE TITLE
amd_fsr_fshack.mypatch failing to be applied

### ISSUE THREAD

**Issue by KokoseiJ** [Frogging-Family/wine-tkg-git#707]
```
 ==> WARNING: Hotfix: Fix for Riot Client updates
 ==> WARNING: Hotfix: Revert Quartz changes breaking Tokyo Xanadu Xe+
 ==> WARNING: Hotfix: Bcrypt fixes for RDR2
 ==> WARNING: Hotfix: Prefer external FAudio for WMA playback
 ==> WARNING: Hotfix: Protonify staging syscall emu
 ==> WARNING: Hotfix: Remove a TRACE in the DXGIGetDebugInterface1 stub causing segfaults.
 -> Hotfixing...
 -> ## Applying reverting hotfix for wine-staging: staging-fa5989b.mystagingrevert
 -> ## Applying hotfix for wine-staging: protonify_stg_syscall_emu-009.mystagingpatch
 -> Applying steam.patch
 -> Applying CSMT-toggle.patch
 -> Applying wine-staging patches... 
     Staging overrides used, if any: -W xactengine3_7-callbacks -W winex11-WM_WINDOWPOSCHANGING -W winex11-_NET_ACTIVE_WINDOW
 -> Applying use_clock_monotonic-staging.patch
 -> Applying fsync-unix-staging.patch
 -> Applying server_Abort_waiting_on_a_completion_port_when_closing_it.patch
 -> Applying fsync_futex_waitv.patch
 -> Applying valve_proton_fullscreen_hack-staging-656edbb.patch
 -> Applying LAA-unix-staging.patch
 -> Applying proton-tkg-staging-86bc819.patch
 -> Applying rdr2.patch
 -> Applying proton-cpu-topology-overrides.patch
 -> Applying proton-vr-831ff10.patch
 -> Applying proton-winevulkan.patch
 -> Applying josh-flat-theme.patch
 -> ## Applying reverting hotfix for plain-wine: f0cd33c.myrevert
 -> ## Applying reverting hotfix for plain-wine: f0cd33c-2.myrevert
 -> ## Applying reverting hotfix for plain-wine: d8be858.myrevert
 -> ## Applying hotfix for plain-wine: 0004-winevulkan1.mypatch
 -> ## Applying hotfix for plain-wine: Return_nt_filename_and_resolve_DOS_drive_path.mypatch
 -> ## Applying hotfix for plain-wine: ef6e33f.mypatch
 -> ## Applying hotfix for plain-wine: 0001-proton-bcrypt_rdr2_fixes4.mypatch
 -> ## Applying hotfix for plain-wine: 0002-bcrypt-Add-support-for-calculating-secret-ecc-keys.mypatch
 -> ## Applying hotfix for plain-wine: a608ef1.mypatch
 -> Found 1 userpatches for plain-wine:
/home/kokoseij/src/wine/wine-tkg-git/wine-tkg-git/amd_fsr_fshack.mypatch
Do you want to install it/them? - Be careful with that ;)
> N/y : y
 -> ######################################################
 -> 
 -> Applying your own plain-wine patch /home/kokoseij/src/wine/wine-tkg-git/wine-tkg-git/amd_fsr_fshack.mypatch
 -> 
 -> ###################
###################################
 ==> ERROR: Patch application has failed. The error was logged to /home/kokoseij/src/wine/wine-tkg-git/wine-tkg-git/prepare.log for your convenience.
 -> Removed BIG_UGLY_FROGMINER - Ribbit
 -> Removed Proton-tkg token - Valve Ribbit
 -> exit cleanup done
```

[customazation.cfg](https://github.com/Frogging-Family/wine-tkg-git/files/8147371/customazation.cfg.txt)
[prepare.log](https://github.com/Frogging-Family/wine-tkg-git/files/8147372/prepare.log)

Game-specific patches that were enabled by default are disabled by me, and it made no changes whether if it's enabled or not.

v7.0 and v7.1 actually fails to apply fsync_futex_waitv patchset, and even with that disabled, it still fails to apply the patch.

I also re-cloned the entire repo just in case my local repo is messed up, with no success.

**Comment by KokoseiJ** [comment]
UPDATE: With staging v7.3 and wine-7.3 branch, it seems that now patch applies without issues. But proton patchset fails to apply, which I think should be the different issue. I'll close this issue for now.

**Comment by Tk-Glitch** [comment]
> v7.0 and v7.1 actually fails to apply fsync_futex_waitv patchset

This will also need fixing since we support building legacy trees. The other issues should be fixed now as long as you're using 7.3. Also as a side note, keep in mind community patches are targeting current HEAD in most cases, so using older versions of wine might prevent some community patches application. Using an older version of the affected patch(es) should work in most cases.
